delft3dcontainermanager/tasks.py

```diff
@@ -52,34 +60,40 @@ def do_docker_create(self, image):
     return container_id
 
 
-@shared_task(bind=True)
+@shared_task(bind=True, throws=(HTTPError))
 def do_docker_start(self, container_id):
     """
-    TODO: implement task do_docker_start
-    This task should start a container with a specific id and return whether
+    Start a container with a specific id and return whether
     the container is started
     """
-    return False
+    client = Client(base_url='unix://var/run/docker.sock')
+    started = client.start(container=container_id)
+    return True if started is None else False
 
 
-@shared_task(bind=True)
-def do_docker_stop(self, container_id):
+@shared_task(bind=True, throws=(HTTPError))
+def do_docker_stop(self, container_id, timeout=10):
     """
-    TODO: implement task do_docker_stop
-    This task should stop a container with a specific id and return whether
+    Stop a container with a specific id and return whether
     the container is stopped
     """
-    return False
+    client = Client(base_url='unix://var/run/docker.sock')
+    stopped = client.stop(container=container_id, timeout=timeout)
+    return True if stopped is None else False
```

Comment:
It seems started is always None, so either:
- always return True (and trust the stop to throw exceptions)
- rewrite to check for container stopped state